### PR TITLE
try an alternate method of plumbing cloud_provider

### PR DESCRIPTION
**Pull request by spiffxp**
also default to '' for controller-manager, it doesn't like auto-detect

### CODE REVIEW

**Comment by spiffxp** [comment]
retest this please

**Comment by spiffxp** [comment]
retest this please

File: terraform/aws/templates/all.tpl [hunk 1]
```diff
@@ -2,7 +2,6 @@
 ansible_ssh_user: core
 ansible_ssh_private_key_file: ${ansible_ssh_private_key_file}
 master_record: ${master_record}
-cloud_provider: aws
```
Comment (maratoid) [inline terraform/aws/templates/all.tpl]: why not just keep it here?

File: terraform/aws/templates/cluster.tpl [hunk 2]
```diff
@@ -23,4 +23,5 @@ logentries_url: ${logentries_url}
 access_port: ${access_port}
 master_private_ip: ${master_private_ip}
 access_scheme: ${access_scheme}
-sysdigcloud_access_key: ${sysdigcloud_access_key}
\ No newline at end of file
+sysdigcloud_access_key: ${sysdigcloud_access_key}
+cloud_provider: aws
```
Comment (maratoid) [inline terraform/aws/templates/cluster.tpl]: and not have to add it here

File: terraform/aws/templates/master.yaml.tpl [hunk 3]
```diff
@@ -45,6 +45,7 @@ write_files:
         master_public_ip: $public_ipv4
         access_scheme: ${access_scheme}
         sysdigcloud_access_key: ${sysdigcloud_access_key}
+        cloud_provider: aws
```
Comment (maratoid) [inline terraform/aws/templates/master.yaml.tpl]: and here

File: terraform/aws/templates/node.yaml.tpl [hunk 4]
```diff
@@ -40,6 +40,7 @@ write_files:
         master_public_ip: ${master_public_ip}
         access_scheme: ${access_scheme}
         sysdigcloud_access_key: ${sysdigcloud_access_key}
+        cloud_provider: aws
```
Comment (maratoid) [inline terraform/aws/templates/node.yaml.tpl]: and here

**Comment by spiffxp** [comment]
`all.tpl` doesn't appear to be rendered on instances, only locally?

**Comment by spiffxp** [comment]
retest this please

**Comment by coffeepac** [comment]
LGTM

**Pull request merged**